code/game/machinery/suit_storage_unit.dm

```diff
@@ -253,29 +251,15 @@
 			src.eject_occupant(usr)
 		src.updateUsrDialog()
 		src.update_icon()
-	/*if (href_list["refresh"])
-		src.updateUsrDialog()*/
 	src.add_fingerprint(usr)
 	return
 
 
 /obj/machinery/suit_storage_unit/proc/toggleUV(mob/user)
-//	var/protected = 0
-//	var/mob/living/carbon/human/H = user
 	if(!src.panelopen)
 		return
-
-	/*if(istype(H)) //Let's check if the guy's wearing electrically insulated gloves
-		if(H.gloves)
-			var/obj/item/clothing/gloves/G = H.gloves
-			if(istype(G,/obj/item/clothing/gloves/yellow))
-				protected = 1
-
-	if(!protected)
-		playsound(src.loc, "sparks", 75, 1, -1)
-		user << "<font color='red'>You try to touch the controls but you get zapped. There must be a short circuit somewhere.</font>"
-		return*/
-	else  //welp, the guy is protected, we can continue
+		
+	else

```

Comment:
sure but there used to be more checks there commented out, which is what I removed